SPECS-EXTENDED/libsamplerate/libsamplerate.spec

```
@@ -0,0 +1,226 @@
+Summary:        Sample rate conversion library for audio data
+Name:           libsamplerate
+Version:        0.2.2
+Release:        4%{?dist}
+License:        BSD
```

Not a blocker, but can you convert this to an [SPDX license expression](https://spdx.org/licenses/)? We're slowly moving away from the Fedora short identifiers. [This tool](https://github.com/spdx/spdx-license-diff) is fairly helpful in finding the exact BSD license variant here.

```suggestion
License:        BSD-2-Clause
```

---

Done. Converted license string to SPDX license expression.